packages/cubejs-schema-compiler/src/compiler/CubeEvaluator.ts

```diff
@@ -66,7 +66,33 @@ export type PreAggregationFilters = {
   scheduled?: boolean,
 };
 
-type PreAggregationDefinition = {
+export type EveryInterval = string;
```

Comment:
Does this need something like a `TODO: Write a real type` comment?

Reply:
YES! Thxn! I'll add it!